### ISSUE TITLE
Service Request by Corinna  Noel

### ISSUE THREAD

**Issue by donriepurcell** [dlounsbu/NYHEALing-IPAtracker-support#59]
- [x] Qualtrics Service Request Form was received from User
- [ ] e-mail acknowledging that User's Qualtrics Service Request Form was sent
- [ ] Problem NOT RESOLVED
- [ ] Problem PARTIALLY RESOLVED 
- [ ] Problem FULLY RESOLVED and Issue is Closed
- [ ] Qualtrics Post-Service Request Survey was sent to User

Use comments feature (below) to document:

1. A clear and concise description of the solution.

2. A clear and concise description of any alternative solutions or features you've considered.

3. Any additional context, such as screenshots about the problem.

**Comment by donriepurcell** [comment]
Hi Jeff,
I investigated this an realized that Corinna is unable to see B3-Retention peers because there still exist a problem in cayuga county as it relates to visually seeing the measures.
Regards 
Donrie 
<img width="1253" alt="Screen Shot 2021-10-15 at 11 19 21 AM" src="https://user-images.githubusercontent.com/80912517/137513213-0d060723-8c00-44e1-952a-a3df540daa47.png">

**Comment by donriepurcell** [comment]
Hi Jeff
As per the county personnel, that data was entered data into the tracker and it was lost when you made a couple of adjustments previously. Can the data be recovered ?

**Comment by dlounsbu** [comment]
Hi Donrie:

I do not think this is a good use of Jeff's time. 

Let's see if we can find the history and use it to inform the edits that they are asking for.

Doing so should help us understand more about how to support the counties. 

Best,
David

**Comment by donriepurcell** [comment]
Jeff simply needs to give the county visual access to their data much the same way he did with the other counties. The data is obviously in the system,

**Comment by jbazinet1111** [comment]
The prior fix was in the investigation of what you described as a very urgent error. Turned out not to be an error at all but rather that no administrator had set up any Other Measures. I explained this in detail more than once. Given that this is the same issue, with the prior issue having a root cause identification, I'm unclear as to why this was assigned to me.

I'd be happy to do this for you, in the exact same fashion that I mentioned above, but I'd need approval for the time involved as it is not an error or a bug, regardless of how much finger-pointing there is. Let me know and I'll send an estimate asap.

**Comment by dlounsbu** [comment]
Hi Jeff:



> The prior fix was in the investigation of what you described as a very urgent error. Turned out not to be an error at all but rather that no administrator had set up any Other Measures. I explained this in detail more than once. Given that this is the same issue, with the prior issue having a root cause identification, I'm unclear as to why this was assigned to me.
> 
> I'd be happy to do this for you, in the exact same fashion that I mentioned above, but I'd need approval for the time involved as it is not an error or a bug, regardless of how much finger-pointing there is. Let me know and I'll send an estimate asap.

I understand what you are writing now. I am working with Donrie now. Will get back to you if we have a problem.

**Comment by dlounsbu** [comment]
Hi Jeff:

So, we explored adding OTHER MEASURES List Options. After adding a new List Option for an OTHER MEASURE for Cayuga, we could not select it and we still did not see any 'tags' (SECTORS, HIGH POPS, etc) for IPA/Smart Strategy.

For Cayuga, in the Data Download, we observed that there was Monitoring Data, but NO Specification Data, for the given IPA/Strategy, and we could see the name of the OTHER MEASURE. 

Per Donrie's point, we think there is a data linking issue. There may also be some confusion on our part about what to do first regarding setting up LIST OPTIONS for Measures, especially OTHER MEASURES.

To summarize, maybe there are two issues - 
1. Reconciling the LIST OPTIONS issue: Setting up all the Other Measures so that we can see them
2. Bringing back IPA/Smart Strategy List Option 'Tags' in the IPA Tracker summary tables (at least for certain IPAs/Smart Strategies  in Cuyuga.

We may need to walk through this via Zoom. 

This is just a particularly messy transition issue.

**Comment by jbazinet1111** [comment]
Yes - the measures that are shown to the user are dependent upon the chosen EPB type. This rule goes back to the old Excel document so we've kept it going throughout.

**Comment by dlounsbu** [comment]
@donriepurcell 

Please follow through on this task for Jeff!!

**Comment by dlounsbu** [comment]
@donriepurcell 

This issue is been open since September 29!

I did not realize that Corrina had been emailing you for assistance. I was not on subsequent emails, only you were.

We cannot let things like this slip between the cracks.

Actually, I assumed, and that issue has been resolved, but not yet closed. I was leaving that to you, Donrie, because you were following up with her.

Donrie, I need you to stay on top of the status of these issues, especially those that are coming directly from coalition members. I try to do this periodically, but you are dedicated to this right now. GitHub makes it easy to see what is done and not done. 

Let’s do better moving forward.

**Comment by donriepurcell** [comment]
This issue became a cantankerous one, which I have tried to address with Jeff to avail. I would appreciate it much if you can take it from here. In other words, I am escalating it at this moment!

**Comment by donriepurcell** [comment]
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40"><p class="MsoNormal" style="margin: 0in 0in 12pt; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><b><span style="font-size: 12pt; color: black;">From:<span class="Apple-converted-space"> </span></span></b><span style="font-size: 12pt; color: black;">Corinna A Noel &lt;can64@cornell.edu&gt;<br><b>Date:<span class="Apple-converted-space"> </span></b>Tuesday, November 9, 2021 at 11:11 AM<br><b>To:<span class="Apple-converted-space"> </span></b>Donrie Purcell &lt;donrie.purcell@einsteinmed.org&gt;<br><b>Cc:<span class="Apple-converted-space"> </span></b>Nicholas Bove &lt;nicholas.bove@einsteinmed.org&gt;, david lounsbury &lt;david.lounsbury@einsteinmed.org&gt;, Nasim Sabounchi &lt;Nasim.Sabounchi@sph.cuny.edu&gt;<br><b>Subject:<span class="Apple-converted-space"> </span></b>Re: Problem resolved !<o:p></o:p></span></p>

CAUTION: This email comes from an external source; the attachments and/or links may compromise our secure environment. Do not open or click on suspicious emails. Please click on the “Phish Alert” button on the top right of the Outlook dashboard to report any suspicious emails.
--


<p class="MsoNormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; color: black;">Thank you, Donrie! <o:p></o:p></span></p><p class="MsoNormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; color: black;"><o:p> </o:p></span></p><p class="MsoNormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; color: black;">Best,<o:p></o:p></span></p><p class="MsoNormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; color: black;">Corinna <o:p></o:p></span></p><div class="MsoNormal" align="center" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; text-align: center;"><span><hr size="0" width="100%" align="center"></span></div><p class="MsoNormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><b><span style="color: black;">From:</span></b><span style="color: black;"><span class="Apple-converted-space"> </span>Donrie Purcell &lt;donrie.purcell@einsteinmed.org&gt;<br><b>Sent:</b><span class="Apple-converted-space"> </span>Tuesday, November 9, 2021 11:07 AM<br><b>To:</b><span class="Apple-converted-space"> </span>Corinna A Noel &lt;can64@cornell.edu&gt;<br><b>Cc:</b><span class="Apple-converted-space"> </span>Nicholas Bove &lt;nicholas.bove@einsteinmed.org&gt;; david lounsbury &lt;david.lounsbury@einsteinmed.org&gt;; Nasim Sabounchi &lt;Nasim.Sabounchi@sph.cuny.edu&gt;<br><b>Subject:</b><span class="Apple-converted-space"> </span>Problem resolved !</span><span><o:p></o:p></span></p><p class="MsoNormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span> <o:p></o:p></span></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;">Hi Everyone,</span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;"> </span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;">It appears that the problem related to the Measures not showing up in Measures Monitoring as described by Corinna has been resolved.<span class="Apple-converted-space"> </span></span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;">It is clear that I had not gone back into the tracker to check and verify if Jeff had fixed the problem subsequent to our GitHub request.<span class="Apple-converted-space"> </span></span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;">The onus to have done so lay completely at my feet.</span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;">Please see attached screen shots of all the previous measures noted in Corinna’s initial email that are now available to be edited.<span class="Apple-converted-space"> </span></span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;"> </span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;">I have also edited B3 RETENTION PEERS within the month of September by adding 10 units as a test subject and it works fine. Thank you everyone for your patience on this matter.<span class="Apple-converted-space"> </span></span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;"> </span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; vertical-align: baseline;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif; color: black;">Regards<br>Donrie Purcell, PhD, MPH</span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; vertical-align: baseline;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif; color: black;"> </span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; vertical-align: baseline;"><b><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif; color: black;">Every generation has the obligation to free men's minds for a look at new worlds,....to look out from a higher plateau than the last generation.</span></b><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif; color: black;"><br>Ellison S. Onizuka <br><br><br>CONFIDENTIALITY: This email (including any attachments) may contain confidential, proprietary and/or privileged information. Any duplication, copying, distribution, dissemination, transmission, disclosure or use in any manner of this email (including any attachments) without the authorization of the sender is strictly prohibited. If you receive this email (including any attachments) in error, please notify the sender and delete this email (including any attachments) from your system. Thank you.</span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;"> </span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;"> </span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;"> </span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;"><span><img width="580" height="262" src="blob:https://github.com/36bd1adc-f90d-4e9a-b0d6-a11eaefd424a" alt="Table

Description automatically generated" data-outlook-trace="F:1|T:1" v:shapes="x_Picture_x0020_8" style="height: 2.729in; width: 6.041in;"></span><span><img width="589" height="185" src="blob:https://github.com/9bdeb11f-c461-451d-a8d9-f96cec6038a6" alt="Graphical user interface, text, application

Description automatically generated" data-outlook-trace="F:1|T:1" v:shapes="x_Picture_x0020_9" style="height: 1.927in; width: 6.135in;"></span></span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;"><span><img width="605" height="218" src="blob:https://github.com/386607f6-23be-45d0-9597-e676ae9e9155" alt="Graphical user interface, application

Description automatically generated" data-outlook-trace="F:1|T:1" v:shapes="x_Picture_x0020_11" style="height: 2.27in; width: 6.302in;"></span></span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;"><span><img width="600" height="319" src="blob:https://github.com/1cf34747-2210-4a10-a6ef-20b478119776" alt="Graphical user interface, application

Description automatically generated" data-outlook-trace="F:1|T:1" v:shapes="x_Picture_x0020_12" style="height: 3.322in; width: 6.25in;"></span></span><o:p></o:p></p><p class="xmsonormal" style="margin: 0in; font-size: 11pt; font-family: Calibri, sans-serif; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="font-size: 12pt; font-family: &quot;Times New Roman&quot;, serif;"> </span><o:p></o:p></p></html>

**Comment by dlounsbu** [comment]
Excellent. Thanks Donrie!
